scripts/t265_to_mavlink.py

```diff
@@ -426,6 +426,9 @@ def realsense_notification_callback(notif):
     global reset_counter
     print("INFO: T265 event: " + notif)
     if notif.get_category() is rs.notification_category.pose_relocalization:
+        if reset_counter >= 255:
+            reset_counter = 1
+
         reset_counter += 1
```

Comment:
Can you move the check after the increment instead? It makes better sense that way.